<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bank</class>
 <widget class="QMainWindow" name="Bank">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_quit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>100</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_authorize">
       <property name="text">
        <string>Authorize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_reject">
       <property name="text">
        <string>Reject</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>351</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="prompt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Double-click a row to select</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
